aten/src/ATen/native/LossCTC.cpp

```diff
@@ -364,6 +365,10 @@ Tensor ctc_loss(const Tensor& log_probs, const Tensor& targets, IntList input_le
 
 // Convenience function accepting Tensors
 Tensor ctc_loss(const Tensor& log_probs, const Tensor& targets, const Tensor& input_lengths, const Tensor& target_lengths, int64_t BLANK, int64_t reduction) {
+
```

Comment:
This, too. (It's probably from me, but I think starting functions with a blank line doesn't match the usual coding style.)